img/e/expressvpn.com.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" class="xv-logo" width="220" height="220"><path class="xv-logo__circle" d="M110 220c60.751 0 110-49.249 110-110S170.751 0 110 0 0 49.249 0 110s49.249 110 110 110z" fill="#c8252c" fill-rule="evenodd"/><path class="xv-logo__outline" fill="#fff" fill-rule="evenodd" d="M38.839 89.579H86.67l4.747 13.357H50.916v14.568h45.682l3.885 10.927H37.849v14.576h67.812l8.735 24.573h24.755l41.134-115.489h-24.602l-28.125 85.471-21.262-62.521H38.839z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" height="149.808" width="186.266"><path d="M48.412 149.808c-20.922 0-34.267-11.754-34.267-28.089 0-5.38 1.594-11.951 7.769-25.3H0l20.12-43.033h140.05c3.187 0 5.377-2.388 5.377-5.175 0-2.792-2.19-5.183-5.378-5.183H46.815l9.962-21.315C62.952 8.566 76.3 0 90.841 0h47.615c27.49 0 47.81 18.924 47.81 43.624 0 29.288-26.894 52.796-57.97 52.796H36.26c-2.793 5.776.395 10.36 7.17 10.36h73.31c12.152 0 21.515 8.564 21.515 19.52 0 12.551-12.15 23.508-25.899 23.508z" style="fill:#da3940;fill-opacity:1"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 186 150"><path fill="#da3940" d="M48 150c-21 0-34-12-34-28 0-6 2-12 8-26H0l20-43h140c3 0 6-2 6-5s-3-5-6-5H47l10-21C63 9 76 0 91 0h47c28 0 48 19 48 44 0 29-27 52-58 52H36c-3 6 1 11 7 11h74c12 0 21 8 21 19 0 13-12 24-26 24z"/></svg>
```

Reply:
This one looks slightly off - maybe removing _all_ decimal places on the numbers was a bit much.

Reply:
Okay how about this one?
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 186.3 149.8"><path fill="#da3940" d="M48.4 149.8c-21 0-34.3-11.7-34.3-28 0-5.5 1.6-12 7.8-25.4H0l20.1-43h140a5.2 5.2 0 1 0 0-10.4H46.9l10-21.3A37.8 37.8 0 0 1 90.8 0h47.7c27.4 0 47.8 19 47.8 43.6 0 29.3-27 52.8-58 52.8h-92c-2.8 5.8.4 10.4 7.1 10.4h73.3c12.2 0 21.6 8.5 21.6 19.5 0 12.6-12.2 23.5-26 23.5z"/></svg>
```